tests/unit/s2n_client_auth_handshake_test.c

```diff
@@ -113,8 +113,8 @@ int s2n_test_client_auth_message_by_message(bool no_cert)
     EXPECT_NOT_NULL(server_config = s2n_config_new());
     EXPECT_NOT_NULL(client_config = s2n_config_new());
     EXPECT_SUCCESS(s2n_config_set_unsafe_for_testing(client_config));
-    EXPECT_SUCCESS(s2n_config_set_signature_preferences(server_config, "20200207"));
-    EXPECT_SUCCESS(s2n_config_set_signature_preferences(client_config, "20200207"));
+    EXPECT_SUCCESS(s2n_config_set_cipher_preferences(server_config, "20190801"));
```

Comment:
for places where s2n_config_set_cipher_preferences() are used, could we add a couple more s2n_connection_set_cipher_preferences() to exercise the connection level configurations too?